### PR TITLE
Add ecologyPlaygroundLoading func to docs

### PR DESCRIPTION
**Pull request by kylecesmat**
Will load placeholder boxes on initial render and fix https://github.com/FormidableLabs/formidable-landers/issues/23

### CODE REVIEW

**Comment by kylecesmat** [comment]
cc @paulathevalley

File: docs/victory-area/docs.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Docs extends React.Component {
           playgroundtheme="elegant"
           customRenderers={appendLinkIcon}
         />
-        <Style rules={VictoryTheme}/>
+        <Style rules={merge(ecologyPlaygroundLoading, VictoryTheme)}/>
```
Comment (paulathevalley) [inline docs/victory-area/docs.js]: Should this be on line 19?

**Comment by paulathevalley** [comment]
1 comment to address, otherwise LGTM!

File: docs/victory-area/docs.js (hunk 1 above)
Comment (kylecesmat) [inline docs/victory-area/docs.js]: YES, good catch this is very wrong

**Pull request merged**